### PR TITLE
Improve text notice during GA set up.

### PR DESCRIPTION
**Pull request by sashadoes**
## Summary

Addresses issue:

- #5648 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/modules/analytics/components/common/ExistingGTMPropertyNotice.js [hunk 1]
```diff
@@ -38,17 +38,17 @@ export default function ExistingGTMPropertyNotice() {
 		select( MODULES_TAGMANAGER ).getSingleAnalyticsPropertyID()
 	);
 
-	if ( ! gtmAnalyticsPropertyID ) {
-		return null;
-	}
+	// if ( ! gtmAnalyticsPropertyID ) {
+	// 	return null;
+	// }
```
Comment (nfmohit) [inline assets/js/modules/analytics/components/common/ExistingGTMPropertyNotice.js:43]: I think this was left out by mistake.

```suggestion
	if ( ! gtmAnalyticsPropertyID ) {
		return null;
	}
```

**Review by nfmohit — CHANGES_REQUESTED** [review]
@sashadoes One little change and we should be good to go, thanks!

**Review by nfmohit — APPROVED** [review]
LGTM 👍 Thanks, @sashadoes!

**Review by techanvil — APPROVED** [review]
LGTM!

**Pull request merged**